### PR TITLE
Added locations to Bohl 2022 (Cambodia) and Wu 2020 (Wuhan). Reran prepare_dashboard.sh

### PR DESCRIPTION
**Pull request by simonleandergrimm**
@jeffkaufman I'm keeping this a draft pull request as I have two open questions:
 - `sample_metadata_classifier.py` got reformatted, even though I'm using black. Is this ok or will I need to fix something on my side here?
 -  I ran `prepare_dashboard.sh` overnight after i) merging the newest commits from main, and ii) changing `sample_metadata_classifier.py` to contain location data for Bohl 2022 and Wu 2020. Now I am unsure if some of the files that were created by `prepare_dashboard.sh` were changed due to i) or ii). Do you have more insights here?

### CODE REVIEW

**Comment by jeffkaufman** [comment]
> sample_metadata_classifier.py got reformatted, even though I'm using black. Is this ok or will I need to fix something on my side here?

The mgs-pipeline repo is not currently using black or any other automatic formatter.  I'd be fine with either:

* Don't auto-format, just make your change to `sample_metadata_classifier.py`
* Keep your auto-formatting, and also set up GitHub actions to require all checkins are black-formatted the way we do for the P2RA repo

**Comment by jeffkaufman** [comment]
> I ran prepare_dashboard.sh overnight after i) merging the newest commits from main, and ii) changing sample_metadata_classifier.py to contain location data for Bohl 2022 and Wu 2020. Now I am unsure if some of the files that were created by prepare_dashboard.sh were changed due to i) or ii). Do you have more insights here?

The safest way to do this is to check out `main`, `git pull`, run `prepare-dashboard-data.sh`, and commit any changes (while probably also poking whoever added data and should have done this already ;)).  Then once that's up to date run it on this branch and see what the changes are.

While `prepare-dashboard-data.sh` is slow the first time you run it (it's syncing a ton of files locally) it should be only a few minutes now.

**Comment by simonleandergrimm** [comment]
> > I ran prepare_dashboard.sh overnight after i) merging the newest commits from main, and ii) changing sample_metadata_classifier.py to contain location data for Bohl 2022 and Wu 2020. Now I am unsure if some of the files that were created by prepare_dashboard.sh were changed due to i) or ii). Do you have more insights here?
> 
> The safest way to do this is to check out `main`, `git pull`, run `prepare-dashboard-data.sh`, and commit any changes (while probably also poking whoever added data and should have done this already ;)). Then once that's up to date run it on this branch and see what the changes are.
> 
> While `prepare-dashboard-data.sh` is slow the first time you run it (it's syncing a ton of files locally) it should be only a few minutes now.

I have now done this, see #53

**Comment by simonleandergrimm** [comment]
@jeffkaufman I'm checking if `prepare-dashboard.sh` works fine through #53. Is this branch good to merge?

**Comment by jeffkaufman** [comment]
@simonleandergrimm It doesn't look like you've run `prepare-dashboard-data.sh` since making this change?  Or if you have you haven't checked in the changes?

**Comment by simonleandergrimm** [comment]
> @simonleandergrimm It doesn't look like you've run `prepare-dashboard-data.sh` since making this change? Or if you have you haven't checked in the changes?

@jeffkaufman I now ran prepare-dashboard.sh.

File: dashboard/comparison_taxids.txt [hunk 1]
```diff
@@ -346,7 +346,6 @@
 149539
 150247
 153135
-154784
```
Comment (jeffkaufman) [inline dashboard/comparison_taxids.txt:349]: This is suspicious, but may not be worth digging into?

**Review by jeffkaufman — APPROVED** [review]

File: dashboard/comparison_taxids.txt (hunk 1 above)
Comment (simonleandergrimm) [reply dashboard/comparison_taxids.txt:349]: I agree.

Comment (simonleandergrimm) [reply dashboard/comparison_taxids.txt:349]: @jeffkaufman Should I create an issue for future reference or leave it?

**Pull request merged**